ui/src/sandbox.rs

```diff
@@ -1082,20 +1082,36 @@ mod test {
     }
 
     #[test]
-    fn formatting_code() {
+    fn formatting_code_edition2015() {
         let req = FormatRequest {
-            code: "fn foo () { method_call(); }".to_string(),
-            edition: None,
+            code: "fn main() { use std::num::ParseIntError; let result: Result<i32, ParseIntError> = try { \"1\".parse::<i32>()? + \"2\".parse::<i32>()? + \"3\".parse::<i32>()? }; assert_eq!(result, Ok(6)); }".to_string(),
+            edition: Some(Edition::Rust2015),
+        };
+
+        let sb = Sandbox::new().expect("Unable to create sandbox");
+        let resp = sb.format(&req);
+
+        assert!(resp.is_err());
```

Comment:
Hmmm, I'm not sure why I even cannot pass the original test case in my local but I think logically this test should be passed 😓? Wait for CI result 🙏